client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -42,7 +44,9 @@ const PendingGappsTosNotice = React.createClass( {
 	},
 
 	getGappsLoginUrl( email, domain ) {
-		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel&continue=https%3A%2F%2Fadmin.google.com%2F${ domain }%2FAcceptTermsOfService%3Fcontinue%3Dhttps%3A%2F%2Fmail.google.com%2Fmail%2Fu%2F1`;
+		return `https://accounts.google.com/AccountChooser?Email=${ email }&service=CPanel' +
```

Comment:
Oh, wow, nice catch - this is what I get for relying just on eslint and syntastic plugin in vim :/